hosting/couchdb/couch/local.ini

```diff
@@ -3,3 +3,6 @@
 [couchdb]
 database_dir = DATA_DIR/couch/dbs
 view_index_dir = DATA_DIR/couch/views
+
+[chttpd_auth]
+timeout = 7200 ; 2 hours in seconds
```

Comment:
Could this not be longer? It's only local, so there should not be any security issue to have it as long as possible

Reply:
This isn't only local, the same image is used in self hosted deployments as well - I wanted to extend it in a way that is useful for users (5 minutes is too short for any real usage IMO) and for our own.